controller.go

```diff
@@ -377,9 +374,15 @@ func (c *Controller) ServeYAML() {
 
 // ServeFormatted serve YAML, XML OR JSON, depending on the value of the Accept header
 func (c *Controller) ServeFormatted(encoding ...bool) {
-	hasIndent := BConfig.RunMode != PROD
-	hasEncoding := len(encoding) > 0 && encoding[0]
-	c.Ctx.Output.ServeFormatted(c.Data, hasIndent, hasEncoding)
```

Comment:
Why not to improve `c.Ctx.Output.ServeFormatted`. All these function implement will move to ctx.

Reply:
It is a hotfix because in the current version it is not working correctly. I will work on the improvement.